### PR TITLE
payments: handle patron payments

### PR DESCRIPTION
**Pull request by BadrAly**
* Removes old Fee resource and its config.
* Adds two new loan fixtures options to create active and paid overdues.
* Closes automatically parent patron transaction when full payment is received.

Co-Authored-by: Aly Badr <aly.badr@rero.ch>

## Why are you opening this PR?

https://tree.taiga.io/project/rero21-reroils/task/1284?kanban-status=1224894

## How to test?

`bootstrap`
`setup`

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?

### CODE REVIEW

File: rero_ils/modules/patron_transactions/api.py [hunk 1]
```diff
@@ -80,11 +81,25 @@ def loan_pid(self):
             return notif.loan_pid
         return None
 
+    @property
+    def document_pid(self):
+        """Return the document pid of the the overdue patron transaction."""
+        from ..notifications.api import Notification
```
Comment (lauren-d) [inline rero_ils/modules/patron_transactions/api.py]: move at top if possible

File: data/loans.json [hunk 2]
```diff
@@ -3,7 +3,8 @@
     "barcode": "reroils1",
     "loans": {
       "active": 2,
-      "overdue": 1,
+      "overdue_active": 1,
```
Comment (rerowep) [inline data/loans.json]: Do we use camel case in JSON files?

File: rero_ils/modules/patron_transactions/api.py (hunk 1 above)
Comment (BadrAly) [reply rero_ils/modules/patron_transactions/api.py]: Not possible

File: data/loans.json (hunk 2 above)
Comment (BadrAly) [reply data/loans.json]: Only in the SON of the resource `document`

**Pull request merged**